### PR TITLE
[TW-1129] OWASP linting rules

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Mentioned that the rule library now includes "Postman's OWASP API guidelines"
* Updated existing instructions to match the new UI changes

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 1]
```diff
@@ -47,13 +47,17 @@ In addition to the rules turned on by default in Postman, you can add other rule
...
+1. Select **Import** to open the rule library.
+1. Select **Import** next to a rule to import it. Details and API format requirements are available under the rule name.
 
-<img alt="Import API Governance rule from Postman library" src="https://assets.postman.com/postman-docs/import-postman-rule-from-rule-library-10.12.0.jpg"/>
+    > You can select **View all** below a set of guidelines to view all of its rules. To import all rules for a particular set of guidelines, select **Import All**.
+
+1. Once you import new rules from the library, you need to [turn them on](#turning-configured-rules-on-and-off) in the API Governance dashboard.
 
-Once you import new rules from the library, you need to [turn them on](#turning-configured-rules-on-and-off) in the API Governance dashboard.
+<img alt="Import API Governance rule from Postman library" src="https://assets.postman.com/postman-docs/import-postman-rule-from-rule-library-10.12.0.jpg"/>
```
Comment (claireriley) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:60]: I think this screenshot might work better under the third step

**Review by claireriley — APPROVED** [review]
LGTM! ⭐ one minor suggestion

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md]: Yes, I agree. This is now updated. Thanks!

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 2]
```diff
@@ -47,13 +47,17 @@ In addition to the rules turned on by default in Postman, you can add other rule
...
+1. Select **Import** to open the rule library.
+1. Select **Import** next to a rule to import it. Details and API format requirements are available under the rule name.
 
-<img alt="Import API Governance rule from Postman library" src="https://assets.postman.com/postman-docs/import-postman-rule-from-rule-library-10.12.0.jpg"/>
+    > You can select **View all** below a set of guidelines to view all of its rules. To import all rules for a particular set of guidelines, select **Import All**.
 
-Once you import new rules from the library, you need to [turn them on](#turning-configured-rules-on-and-off) in the API Governance dashboard.
+    <img alt="Import API Governance rule from Postman library" src="https://assets.postman.com/postman-docs/import-postman-rule-from-rule-library-10.12.0.jpg"/>
+
+1. Once you import new rules from the library, you need to [turn them on](#turning-configured-rules-on-and-off) in the API Governance dashboard.
```
Comment (savageCoder55) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:60]: I would recommend reframing this to suggest that after importing, a rule needs to be added to a workspace group in order to enforce it for the specific workspaces in the workspace group.

**Review by savageCoder55 — APPROVED** [review]
One small comment. Nothing major.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md]: Thanks! This is now updated. I also updated the final step in the "Adding custom rules" section to include similar details. I updated the step in the following PR/commit: https://github.com/postmanlabs/postman-docs/pull/4814/commits/fe378c526c2b14ebe61cb15e1369bc5475bf2e0e

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**